requirements-dev-sqlalchemy.txt

```diff
@@ -18,5 +18,5 @@ pyodbc>=4.0.30  # sqlalchemy_tests
 snowflake-connector-python==2.3.8  # sqlalchemy_tests
 # END NOTE
 snowflake-sqlalchemy>=1.2.3  # sqlalchemy_tests
-sqlalchemy>=1.3.16,<1.4  # sqlalchemy_tests
+sqlalchemy>=1.3.16<1.4.0  # sqlalchemy_tests
```

Comment:
I think this might be a typo, and I'm excited we can just fix the underlying issue and no longer need to pin to an older version of SQL alchemy!

Reply:
this is a typo. I'm still testing whether this is compatible with both <1.4.0 and > 1.4.0

Reply:
👍